python/ray/train/session.py

```diff
@@ -501,3 +504,50 @@ def train_func():
     if session is None:
         return 1
     return session.world_size
+
+
+class SessionMisuseError(Exception):
+    """Method or function was used outside of a session."""
+
+
+def _raise_accelerator_session_misuse():
+    """Raises a SessionMisuseError because a utility function was used improperly."""
+    raise SessionMisuseError(
+        "Utility functions should be called inside a training function executed by "
```

Comment:
Should we be more specific here?

```suggestion
        "prepare/accelerate utility functions should be called inside a training function executed by "
```